packages/itwinui-css/src/utils/notification-marker/notification-marker.scss

```diff
@@ -2,25 +2,38 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 
 $iui-notification-marker-border-width: var(--iui-size-3xs);
+$iui-notification-before-size: calc(var(--iui-size-xs) + $iui-notification-marker-border-width * 2);
+$iui-notification-before-size-radius: calc($iui-notification-before-size * 0.5);
```

Comment:
Can you explain what these variables mean and how they help in positioning? The names are not very clear.

Reply:
Ohh, yes, I think the names are not clear. I added comments and also changed the names as follows:
* `$iui-notification-marker-border-width` -> `$iui-notification-marker-spacing` (since now there is no border)
* `$iui-notification-before-size` -> `$iui-notification-marker-size`
* `$iui-notification-before-size-radius` -> `$iui-notification-marker-size-radius`

Do you recommend some other names? Also, do you think the comments are needed or the names are descriptive enough?

Reply:
Comments shouldn't be necessary if the names are descriptive. But even with your explanations, I don't see how these variables help with positioning/sizing.

1. The `width` and `height` of the `::before` element can just be `var(--iui-size-xs)`, replacing the need for marker-size and marker-spacing.
2. The -radius variable is not used directly (it has more math applied on it), so I'm not sure what's even the purpose of it.
![](https://user-images.githubusercontent.com/9084735/192568926-b46044a4-3b89-4506-be45-50d9310f39b3.png)

Reply:
After calculating the actual values of the math calculations, some of them turned out to be 0. Also, some calculations were unnecessary (like the `width` and `height` you mentioned in point 1).

I simplified those calculations, tried to use more descriptive variable names, and also limited to only using variables that I felt were useful.